record_tx.go

```diff
@@ -100,15 +100,15 @@ func waitForRecordTxWriteLock(ctx context.Context, c ClientInterface, key string
 		)
 		if err == nil {
 			// TODO: change to DEBUG level log when we will support it
-			c.Logger().Info(ctx, fmt.Sprintf("added write lock %s", lockKey))
+			c.Logger().Info().Msgf("added write lock %s", lockKey)
```

Comment:
Is it possible to use `zerolog.Debug` here ?

Reply:
Done